File: Labda.java
public class Hellow {


public static void main(String[] args) throws InterruptedException {

     int xj = 0, xk = 0, yj = 0, yk = 0;
     int row = 80 * 2, col = 24 * 2;

char escCode = 0x1B;

    for (;;)
     {
         xj = (xj - 1) % row;
         xk = (xk + 1) % row;

         yj = (yj - 1) % col;
         yk = (yk + 1) % col;



	System.out.print(String.format("%c[%d;%df",escCode,0,0)+"--------------------------------------------------------------------------------");

	System.out.print(String.format("%c[%d;%df",escCode,24,0)+"--------------------------------------------------------------------------------");

	System.out.print(String.format("%c[%d;%df",escCode,Math.abs ((yj + (col - yk)) / 2),Math.abs ((xj + (row - xk)) / 2))+'X');

	Thread.sleep(250);
        System.out.print("\033\143");

  System.out.print("\033[H\033[2J"); 
  		}
	}


}

File: Labda.py
import sys
from time import sleep
import os
x, y = 1, 1
xk, yk= x, y
row = 80
col = 26
def print_there(x, y, text):
    sys.stdout.write("\x1b7\x1b[%d;%df%s\x1b8" % (x, y, text))
    sys.stdout.flush()
while True:
    x = x + (-1)**((xk - 1)/(row - 3))
    y = y + (-1)**((yk - 1)/(col - 3))

    xk = xk + 1
    yk = yk + 1

    for i in range(1, 80):
	print_there(1, i, "-")
	print_there(24, i, "-")
    print_there(y, x, "X")
    sleep(0.1)
    os.system('cls||clear')
